Add files via upload

metavaults opened the pull request.

metavaults committed:
Add files via upload

sacdallago commented on scrapeIDArtists/server.js:
```diff
@@ -0,0 +1,86 @@
+    table: []
+  };
+
+  // obj.table.push({id: 1, type:"person", name:"Lukas Music Star", country: "LT"});
```
Why is there commented code in this PR? What's the purpose of this code, why is it not uncommented, is it needed?

sacdallago replied:
If the sense is to have an example of what the data looks like, use the long comments: `\** ....` and describe what you are doing

kordianbruck commented on scrapeIDArtists/server.js:
```diff
@@ -0,0 +1,86 @@
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
```
You can run a .js file with node without having to start a http server. If you are only scraping data why do you need a webserver?

kordianbruck commented on scrapeIDArtists/server.js:
```diff
@@ -0,0 +1,86 @@
+            request({
+                url: url,
+                headers: {
+                'User-Agent': 'ClassicalMusic/1.5.0 ( laucius@gmail.com )'
```
Be careful with commiting your email - spamers are crawling Github to find those

TimHenkelmann committed:
- removed artists.json output and package.json
- added .gitignore file and artists_example.json
- updated server.js according to tutor input:
    - removed server
    - removed unnecessary comments
    - removed email from request

TimHenkelmann commented:
Applied tutor suggestions for improvement:
 - removed artists.json output and package.json
- added .gitignore file and artists_example.json
- updated server.js according to tutor input:
    - removed server
    - removed unnecessary comments
    - removed email from request

Ready for review!

TimHenkelmann commented:
Hey @sacdallago we implemented your improvement proposals (removing unnecessary comments etc.) but it still says that that changes are requested. Could you help us with that?

kordianbruck commented:
@TimHenkelmann he probably has not yet checked his emails. I've dismissed his review. You are ready to merge.

sacdallago commented:
Thanks @kordianbruck :D sorry @TimHenkelmann That's precisely why we tutors should not be handling PRs :) But we do it here just to give you a hand, don't hate us ❤️
